src/DiscoveryServerManager.cpp

```diff
@@ -1400,18 +1399,22 @@ void DiscoveryServerManager::loadSubscriber(
     // default topic is the static HelloWorld one
     const char* profile_name = sub->Attribute(DSxmlparser::PROFILE_NAME);
 
-    SubscriberAttributes subatts;
+    ///SubscriberAttributes subatts;
+    SubscriberQos subqos;
```

Comment:
You are correct. This is neither used nor configured in the XML files (there are no `<subscriber>/<data_reader>` profiles. Only participant profiles are found in the XML configuration files, so it can be safely removed and only worry about getting the TopicAttributes in order to create the test events.